x-pack/plugins/apm/server/lib/rum_client/get_js_errors.ts

```diff
@@ -23,7 +19,7 @@ export async function getJSErrors({
   pageSize,
   pageIndex,
 }: {
-  setup: Setup & SetupTimeRange & SetupUIFilters;
+  setup: Setup & SetupTimeRange;
```

Comment:
SetupUIFilters is moved into Setup type?

Reply:
Correct. `SetupUIFilters` only defined the `uiFiltersES` property, which now defaults to an empty array on the core `Setup` object, so it's not necessary to define `SetupUIFilters` any longer.